### ISSUE TITLE
group mentoring rescope/process improvements

### ISSUE THREAD

**Issue by parispittman** [kubernetes/community#2659]
After conducting a large test and a retro, we discovered the following that needs to be corrected before a full scale launch can happen. This issue will get rolling updates and edits after we address each one of these items on group-mentoring.md doc. 

- [ ] learning / workshops / curriculum 
we should make a one stop doc in the mentoring folder that has all of the video links/playlists and other resources to step by step contribution process and role based skill trainings that we already have. 

see other related open ended questions section on the group-mentoring.md doc

- [ ] think about the slack mentoring groups structure. 
current state - private slack channel is created by cohort. 

should we create larger private groups for "future reviewer", "future approver" to capitalize on peer mentoring or is this redundant/another channel/not set a good precedence and direct everyone talk in k-dev? 
Pros of larger private groups for mentoring - people will want to ask more questions because the signal to noise ratio is smaller, will be ok publicly asking small questions and/or limit the "sorry if im asking this but..." that is heard often in learning situations. We can still keep the smaller cohort channels too? I could be overthinking this but I've noticed when the channel is bigger (but not huge - like that of #meet-our-contributors), everyone is sharing the burden to help out.

**Comment by markyjackson-taulia** [comment]
cc @helayoty @chris-short 
Going to mover this to `In Progress` and start working through this. I foresee some PR in the near future

**Comment by alejandrox1** [comment]
proposal that I made for the group on a zoom call:

Besides actual approvers for an area also have as mentors people who are comfortable with the code base.
Not necessarily approvers but people who can help others interested in becoming approver for SIG X.
Mentors should be able to help with reviews, coding, technical knowledge but also help create work.
Creating work in this context would be to look for places that need improvements, things that need fixing, etc. Something that a contributor can create an issue about and begin to work (with the intent of expand from only "help wanted" issues).

The goal would be that if people can work on an area enough, they will naturally get the opportunity to collaborate with approvers in an area.
The more PRs they write, the more they review, the more issues they create and triage will help a mentee and area approver(s) work together.
Based on that members of the group could request approvers for the area they have been working on to assign them as reviewers/approvers.

tl;dr enable people to do the work that an approver does and provide the means to help them climb the contributor latter as they become acquainted with certain areas of the project.
More open ended collaboration: mentee has to work with mentors to "create work" that will give them a track record.

Like everything in this Kubernetes :world_map:, any and all feedback/criticism welcomed :-)

**Comment by markyjackson-taulia** [comment]
@jberkus  During the last mentoring sub-project meeting you and @parispittman  were assigned this issue. Is it ok if I assign it to you? I didn't want to do that without your approval 😺

**Comment by mrbobbytables** [comment]
Did we make some progress on this? Just circling back and checking in^^:;;;

**Comment by parispittman** [comment]
/close

bullets:
1 - we are adding more to the curriculum structure but not as in depth as listed here so we went in another lane of the same road :) 
2 - we have a #pr-reviews slack channel, #contributors slack, and a bunch of other solutions now
